src/query/storage/converter_test.go

```diff
@@ -275,3 +276,128 @@ func BenchmarkFetchResultToPromResult(b *testing.B) {
 		benchResult = FetchResultToPromResult(fr, false)
 	}
 }
+
```

Comment:
Can we have a test that converts from annotations to attributes then back to annotations, then back to attributes to ensure the fields get set correctly?

Reply:
Hmm, would be nice, but there is no such round-trip conversion. The conversion flow goes like this:
API protobuf -> attributes -> annotation payload protobuf